lib/ulid/rails.rb

```diff
@@ -12,10 +12,9 @@ module Rails
     extend ActiveSupport::Concern
 
     class_methods do
-      def ulid(column_name, primary_key: false, auto_generate: nil)
+      def ulid(column_name, auto_generate: false)
         attribute column_name, ULID::Rails::Type.new
 
-        auto_generate = primary_key || auto_generate
```

Comment:
Wait, so… `primary_key` and `auto_generate` were the same thing? And `primary_key: true, auto_generate: false` would still be auto-generating?

Reply:
yes, nothing can stop a primary key from not generating 😬